src/Confluent.Kafka/Consumer.cs

```diff
@@ -1128,8 +1128,8 @@ public Task<CommittedOffsets> CommitAsync(IEnumerable<TopicPartitionOffset> offs
             => kafkaHandle.CommitAsync(offsets);
 
         /// <include file='include_docs.xml' path='API/Member[@name="Consumer_Seek"]/*' />
-        public void Seek(TopicPartitionOffset tpo, TimeSpan timeout)
-            => kafkaHandle.Seek(tpo.Topic, tpo.Partition, tpo.Offset, timeout.TotalMillisecondsAsInt());
+        public void Seek(TopicPartitionOffset tpo)
+            => kafkaHandle.Seek(tpo.Topic, tpo.Partition, tpo.Offset, 1000);
```

Comment:
I think it will be better to use an infinite timeout (-1) here, that'll make the call behave consistently regardless of load.

Reply:
yep, sounds good.